lua/methods/coalitions.lua

```diff
@@ -21,13 +21,158 @@ local skill = {
   Player = 5
 }
 
+-- types for aircraft taking off
+local takeoffTypes = {
+  [1] = "TakeOff",
+  [2] = "TakeOffParking",
+  [3] = "TakeOffParkingHot",
+  [4] = "Turning Point",
+  runway = "TakeOff",
+  cold = "TakeOffParking",
+  hot = "TakeOffParkingHot",
+  air = "Turning Point"
+}
+
+-- actiions for aircraft taking off
+local takeoffActions = {
+  [1] = "From Runway",
+  [2] = "From Parking Area",
+  [3] = "From Parking Area Hot",
+  [4] = "Turning Point",
+  runway = "From Runway",
+  cold = "From Parking Area",
+  hot = "From Parking Area Hot",
+  air = "Turning Point",
+}
+
 --local altitudeType = {
 --  [1] = "BARO",
 --  [2] = "RADIO",
 --  BARO = 1,
 --  RADIO = 2
 --}
 
+local buildAirplanePoints = function(points)
+  local builtPoints = {}
+  for _, pointData in pairs(points) do
+    local pointVec3
+    if type(pointData.place) == "string" then
```

Comment:
I'll think about if we can do this in a different way by having gRPC types. No need to change this for the moment, just an FYI.

If may be that we do not allow clients to specify anything other than a Lat/Lon/Alt point since the client can get the information on airfield locations and zones themselves and send us that data instead of making us find it server-side.

Reply:
On reflection I think we should only allow Lat/Lon/alt using the `Position` Protobuf message. Clients should be able to get all the information they need to be able to popualte this information themselves and it simplifies the lua a lot.